game/src/js/scenes/TitleScene.js

```diff
@@ -41,23 +41,49 @@ export default class TitleScene extends BaseScene {
     });
 
     //controls, credits, offline mode buttons
-    let settingsButton = new Button(this, 450, 350, 'SETTINGS');
-    settingsButton.buttonDown(() => {
-      new DOMModal(this, 'settings', {
-        cancelButtonSelector: '.exit',
-        onCancel: (modal) => {
-          modal.close();
-        }
+    if(ServiceWorker.isSupported()) {
+      const assets = this.cache.json.get('assets');
+      let offlineMode = localStorage.getItem('offlineStatus');
```

Comment:
I don't think we need to store this in local storage, since we can check to see if offline mode is enabled by virtue of the SW being installed (as the old checkbox was).

Reply:
This means you don't need lines 46-51